docs/sources/get-started/quick-start.md

```diff
@@ -194,165 +199,337 @@ You can view your logs using the command line interface, [LogCLI](/docs/loki/<LO
    - [Builder mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#builder-mode), which provides a visual query designer.
    - [Code mode](https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#code-mode), which provides a feature-rich editor for writing LogQL queries.
 
-   Next we’ll walk through a few simple queries using both the builder and code views.
+   Next we’ll walk through a few queries using the code view.
 
 1. Click **Code** (3) to work in Code mode in the query editor.
 
-   Here are some sample queries to get you started using LogQL.
-   These queries assume that you followed the instructions to create a directory called `evaluate-loki`.
-
-   If you installed in a different directory, you’ll need to modify these queries to match your installation directory.
+   Here are some sample queries to get you started using LogQL. After copying any of these queries into the query editor, click **Run Query** (4) to execute the query.
 
-   After copying any of these queries into the query editor, click **Run Query** (4) to execute the query.
-
-   1. View all the log lines which have the container label `evaluate-loki-flog-1`:
+   1. View all the log lines which have the `container` label value `greenhouse-main_app-1`:
       <!-- INTERACTIVE copy START -->
       ```bash
-      {container="evaluate-loki-flog-1"}
+      {container="greenhouse-main_app-1"}
       ```
       <!-- INTERACTIVE copy END -->
       In Loki, this is a log stream.
 
       Loki uses [labels](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/) as metadata to describe log streams.
 
       Loki queries always start with a label selector.
-      In the previous query, the label selector is `{container="evaluate-loki-flog-1"}`.
+      In the previous query, the label selector is `{container="greenhouse-main_app-1"}`.
 
-   1. To view all the log lines which have the container label `evaluate-loki-grafana-1`:
```

Comment:
I'm sad that we've lost these other examples in the revision.  I was trying to build in some simple LogQL lessons with them.

Reply:
hmm, let me work on that. I wonder if we can still achieve the same with the containers at our disposal.